main.js

```diff
@@ -0,0 +1,114 @@
+import './style/style.sass'
+import './style/reset.css'
+import $ from 'jquery'
+import { v4 as uuidv4 } from 'uuid'
+
+const inputText = $('.input-text')
+const buttonAddTodo = $('.add-item')
+const todoList = $('.todo__list')
+
+const filtrAll = $('.filtr-all')
+const filtrTodo = $('.filtr-todo')
+const filtrDone = $('.filtr-done')
+
+let todoArray = []
+
+const createNewTodo = getText => {
+    
+    const todo = $('<li>', {
+        class: 'todo__item',
+        text: getText,
+        'data-id': uuidv4()
+    })
+
+    return todo
+}
+
+inputText.keyup(event => event.keyCode === 13 ? addTodo() : null)
+
+buttonAddTodo.click(() => addTodo())
```

Comment:
proxy, I don't know if you know that:
```
() => addTodo()
```
is equal to:
```
addTodo
```
You don't need an additional arrow function to make it work, so please use:
```
buttonAddTodo.click(addTodo)
```